js/app.js

```diff
@@ -0,0 +1,72 @@
+console.log('yo')
+
+// Data types
+// • light switch on or off: boolean
+// • string- "joeblah@blah.com"
+// • object- holds all of the spaceships info
+// • array
+// • array & object in array
+// • array & object in array
+
+// Git commit "selecting data types done"
+
+// Take it Easy
+
+// let myArr = ['blue', 'yellow', 'green', 'orange', 'purple', 'red'] 
+// let color1 = myArr[0];
+
+// let ryneL = {
+// 	favoriteFood: "pizza",
+// 	hobby: "playing with my dog",
+// 	current city: "chi town",
+// 	fav data type: "array"
+// };
+
+// ryneL[hobby]
+
+// "created data structures and accessed it"
+
+
+// Crazy Object
+
+const crazyObject = {
+  taco: [{meat: 'steak',
+         cheese: ['panela', 'queso', 'chihuahua']},
+         {meat: 'chicken',
+          salsa: ["pico", "hot", "hotter", "really hot", "really really hot", "omg my mouth is burning"]},
+        ],
+  larry: {
+    nicknames: ["LD", "Chicken Teriyaki Boyyyyyy"],
+    quotes: ["Pretty pretty prettayyyyy good", "Is that a parkinson's thing?", "women love a self confident bald man", "I'm a walking talking enigma"],
+    characters: [{
+      name: "Jeff",
+      occupation: "manager"
+    },
+    {
+      name: "funkhauser",
+      occupation: "tv dude"
+    },
+    {
+      name: "susie",
+      occupation: "jeffs wife",
+      favourtieHobby: "Swearing at Larry and Jeff"
+    },
+    ]
+  }
+}
+
+• console.log(crazyObject(taco.salsa[5]))
+• console.log(crazyObject(larry.quotes[0]))
+• console.log(crazyObject(larry.characters.favourtieHobby))
+• console.log(crazyObject(larry.nicknames[1]))
+
```

Comment:
**1.5/5**
omitted object printout.  syntax needs work.  
I had to edit pretty heavily to get what you had to work:
```javascript
console.log(crazyObject['taco'][1].salsa[5])
console.log(crazyObject['larry'].quotes[0])
console.log(crazyObject['larry'].characters.favourtieHobby)
console.log(crazyObject['larry'].nicknames[1])
```